gluten-core/src/main/scala/io/glutenproject/expression/UnaryExpressionTransformer.scala

```diff
@@ -60,6 +58,35 @@ class NormalizeNaNAndZeroTransformer(
   }
 }
 
+class PromotePrecisionTransformer(child: ExpressionTransformer, original: PromotePrecision)
+  extends ExpressionTransformer with Logging {
+
+  override def doTransform(args: java.lang.Object): ExpressionNode = {
+    child.doTransform(args)
+  }
+}
+
+class CheckOverflowTransformer(child: ExpressionTransformer, original: CheckOverflow)
```

Comment:
add a parameter: `substraitExprName`, see the example `String2TrimExpressionTransformer`

Reply:
fix